packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -167,6 +167,13 @@ export default {
         }
     },
 
+    beforeDestroy () {
+        // In case modal is destroyed without being closed first, re-enable scrolling
+        const { megaModalDocument } = this.$refs;
```

Comment:
Probably no need to descructure the `megaModalDocument` here as it's not used anywhere else. You could just do `enableBodyScroll(this.$refs.megaModalDocument);`?